### PR TITLE
Fix: Bump Support Versions

### PR DESCRIPTION
**Pull request by brolewis**
Bump the versions for pre-commit-config to get to work with Python 3.8
(mypy) and new path for flake8. Also updated the lock file for docutils
error.

### CODE REVIEW

File: jiratools/__init__.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def _list_from_config(key_name: str) -> List[str]:
     _load_config()
     if not CONFIG:
         raise ConfigNotFoundException
-    return list(filter(None, [x.strip() for x in CONFIG.get(key_name, "").split(",")]))
+    return list(filter(None, (x.strip() for x in CONFIG.get(key_name, "").split(","))))
```
Comment (bradsbrown) [inline jiratools/__init__.py:180]: Thanks `flake8-tuple`, I guess? 😆

**Review by bradsbrown — APPROVED** [review]

File: jiratools/__init__.py (hunk 1 above)
Comment (brolewis) [reply jiratools/__init__.py:180]: Yeah, I mean, _technically_ it would be more efficient at a large scale?

**Review by dgou — APPROVED** [review]
LGTM. Simple change.
I'm not fan of mypy but this looks ok to me.

File: jiratools/__init__.py (hunk 1 above)
Comment (dgou) [reply jiratools/__init__.py:180]: Nah, that's a generator expression https://docs.python.org/3/reference/expressions.html#generator-expressions
(the parenthesis around generator expressions are required except in a few instances)

**Review by dgou — APPROVED** [review]
LGTM.

**Pull request merged**